client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -157,42 +196,65 @@ function SupportText( { hasPlanInCart, isJetpackNotAtomic } ) {
 	return <span>{ translate( 'Email support' ) }</span>;
 }
 
-function CheckoutSummaryFeaturesListDomainItem( { domain } ) {
+function CheckoutSummaryFeaturesListDomainItem( { domain, isMonthlyPricingTest, hasMonthlyPlan } ) {
 	const translate = useTranslate();
+	const bundledText = isMonthlyPricingTest
+		? translate( 'free for one year' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Free domain for one year' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/monthly-pricing-abtest)